WcaOnRails/config/locales/en.yml

```diff
@@ -1665,72 +1665,89 @@ en:
         title: "How do I obtain a WCA ID and a WCA profile?"
         #context: Answer to "How do I obtain a WCA ID..."
         content:
-          '1': "If you have never competed before, go to a <a href='/competitions'>competition</a>!  A WCA ID and WCA profile will be created for you when the results from your first competition are published."
+          '1': "The only way, if you have never competed before, is to go to a <a href='/competitions'>competition</a>! A WCA ID and WCA profile will be created for you when the results from your first competition are published."
       '2':
         #context: Answer to "How can I find a WCA competition"
         title: "How can I find a WCA competition?"
         content:
-          '1': "Check out our list of <a href='/competitions'>competitions</a>"
+          '1': "Check out our list of <a href='/competitions'>competitions</a>. In the region drop down menu, you can select your country to make the browsing process easier. Make sure to check back often as new competitions are announced everyday!"
       '3':
         #context: Answer to "How can I register for a competition? ..."
         title: "How can I register for a competition? Who can I refer to if I have problems registering for a competition?"
         content:
-          '1': "Many competitions do registration right here on the WCA website, but some use their own systems. You should contact the organizers of the competition you want to compete in for more details. %{comp_search_form}"
+          '1': "Many competitions do registration right here on the WCA website, but some use their own systems. You should contact the organizers of the competition you want to compete in for more details. You can find the contact information for the organizers on the competition’s website or choose the competition you are attending in our <a href='/contact/website'>Contact Form</a>.  %{comp_search_form}"
       '4':
-        title: "What are the requirements for attending a WCA competition? What do I need to know before attending a WCA competition?"
-        #context: Answer to "What are the requirements for attending a WCA competitions? ..."
+        title: "What are the requirements for attending a WCA competition?"
```

Comment:
It's a generic comment for your changes, but please be aware that changing the question's title will actually change its permalink (eg: if you used the current [link to that entry](https://www.worldcubeassociation.org/faq#what-are-the-requirements-for-attending-a-wca-competition-what-do-i-need-to-know-before-attending-a-wca-competition) somewhere, it won't be valid anymore after these changes).